python/run-tests.py

```diff
@@ -161,7 +157,7 @@ def run_individual_python_test(target_dir, test_name, pyspark_python):
 
 
 def get_default_python_executables():
-    python_execs = [x for x in ["python3.6", "python2.7", "pypy"] if which(x)]
```

Comment:
pypy?

Reply:
We should install `pypy3` and add it back here as well. I will also comment and file a separate JIRA.